Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -193,7 +193,7 @@ void RenderTheme::adjustStyle(RenderStyle& style, const Element* element, const
     auto autoAppearance = autoAppearanceForElement(style, element);
     auto appearance = adjustAppearanceForElement(style, element, autoAppearance);
 
-    if (appearance == StyleAppearance::None)
+    if (appearance == StyleAppearance::None || appearance == StyleAppearance::Base)
```

Comment:
We should consider adding a helper in `StyleAppearance.h` for this at some point, since this check effectively shows up in 4 places.